### ISSUE TITLE
Index is reporting itself as up to date but is actually empty

### ISSUE THREAD

**Issue by Themanwithoutaplan** [Ascoware/get-iplayer-automator#467]
*What steps will reproduce the problem? Be specific, and provide as much detail as possible.*

Start the app. Search is immediately available but cannot find any programme. Forcing an index an update has no effect.

*What is the expected output? What do you see instead?*

*What version of GiA are you using? On what version of OS X?*

GiA 1.2.6 on 10.15.7

*Are you using a web proxy? DNS proxy? VPN? VPS?*

Yes, Ivacy VPN but the issue is the same with or without it.

*What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.** *

Gone Fishing

*Please provide any additional information below.*

The problem might be related to the most recent version. Downloaded 1.2.6 yesterday and it's not worked since.
The log, even in verbose mode, is not very informative. IPlayer is working fine in the browser. 

```
Updating Programme Index Feeds...

BBC index was up to date.
Updating Programme Index Feeds...

BBC index was up to date.
```

**Comment by Themanwithoutaplan** [comment]
I guess that the real problem is in get_iplayer itself. I tried 3.3.4 from MacPorts and it's complaining about something duplicate. Downgrading to 1.25.4 resolves my immediate problem.

**Comment by Themanwithoutaplan** [comment]
I suspect this issue in get_iplayer is the culprit https://github.com/get-iplayer/get_iplayer/issues/457

**Comment by PeteSut5939** [comment]
I have same problem, again following update to latest version last week
same log report as Themanwithoutaplan
all searches fail
"Use Current Webage" (BBC R4) Status: Not Available,  Shew: Retrieving Metadata...
Sadly, do not know how to roll back to previous version

**Comment by Themanwithoutaplan** [comment]
You can get previous releases from the Releases page: https://github.com/Ascoware/get-iplayer-automator/releases. No idea why it's not in the navigation.

**Comment by skovatch** [comment]
> I suspect this issue in get_iplayer is the culprit [get-iplayer/get_iplayer#457](https://github.com/get-iplayer/get_iplayer/issues/457)

The get_iplayer bug referenced only shows up when you have perl warnings turned on, and it's only complaining about a repeated option flag. What does `get_iplayer --refresh --verbose` give you with get_iplayer 3.34?

**Comment by Themanwithoutaplan** [comment]
```sh
Duplicate specification "downloadabortonfail|download-abort-onfail|download-abort-onfail!" for option "download-abort-onfail"
Duplicate specification "downloadabortonfail|download-abort-onfail|download-abort-onfail!" for option "download-abort-onfail"
get_iplayer v3.34
  Copyright (C) 2008-2010 Phil Lewis, 2010- get_iplayer contributors
  This program comes with ABSOLUTELY NO WARRANTY; for details use --warranty.
  This is free software, and you are welcome to redistribute it under certain
  conditions; use --conditions for details.

INFO: Start: 2024-01-12T11:15:49 (1705054549)
INFO: encodinglocale = UTF-8
INFO: encodinglocalefs = UTF-8
INFO: encodingconsoleout = UTF-8
INFO: encodingconsolein = UTF-8
INFO: encodingwebrequest = UTF-8
INFO: ${^UNICODE} = 0
INFO: Profile dir: /Users/charlieclark/.get_iplayer
INFO: User options file: /Users/charlieclark/.get_iplayer/options
INFO: System options file: /opt/local/etc/get_iplayer/options
-==--==--==--==--==--==--==--==--==--==--==--==--==--==--==--==--==--==--==--==-
Current options:
	encodingconsolein = UTF-8
	encodingconsoleout = UTF-8
	encodinglocale = UTF-8
	encodinglocalefs = UTF-8
	encodingwebrequest = UTF-8
	refresh = 1
	verbose = 1

INFO: Search args: ''
INFO: No file cache exists for tv
INFO: Cache refresh limit (tv): 2023-12-13T10:15:49+00:00
INFO: Will refresh channel BBC Alba
INFO: Will refresh channel BBC Four
INFO: Will refresh channel BBC News
INFO: Will refresh channel BBC One
```

**Comment by PeteSut5939** [comment]
thanks Themanwithoutaplan, 
previous version downloaded & working fine

**Comment by skovatch** [comment]
Should be fixed in v1.26.1.

**Comment by Themanwithoutaplan** [comment]
Works great, thanks!
